spiceaidocs/docs/components/embeddings/index.md

```diff
@@ -7,12 +7,12 @@ pagination_prev: null
 pagination_next: null
 ---
 
-Embedding models are used to convert raw text into a numerical representation that can be used by machine learning models.
-
-Spice supports running embedding models locally, or use remote services such as OpenAI, or [la Plateforme](https://console.mistral.ai/).
+Embedding models convert raw text into numerical representations that can be used by machine learning models. Spice supports running embedding models locally or using remote services such as OpenAI or [la Plateforme](https://console.mistral.ai/).
```

Comment:
Is `la Plateforme` correct?

Reply:
Yes, it is correct https://mistral.ai/news/la-plateforme/